src/reporter/reporter.py

```diff
@@ -90,7 +91,20 @@ def _validate_payload(payload):
 
     if 'id' not in payload:
         return 'Entity id is required in notifications'
-
+     # NGSI-LD temporal property
```

Comment:
the implementation should be in `reporter.timex` not here I think?

Reply:
Hi @c0c0n3 , I tried implementation in `reporter.timex`  but I faced some issues and after debugging code as per my understanding if `modifiedAt` or `observedAt` is in payload then it should be validated first so I implemented in `reporter.py` file. 
Please let me know if I am missing something. Thank you

Reply:
@daminichopra maybe the issue was not clear.

each time a payload arrives, a time_index is computed with a specific logic in `reporter.timex`, following the introduction of support for ngsi-ld, this logic needs to be updated.

so I am not sure that the PR so far complies with that.